README.md

```diff
@@ -1,208 +1,262 @@
-- The index.html and the components.json file must include the following reference to this javascript library
-`<script type="text/javascript" src="widgets/pushNotifications/lib/PushNotification.js"></script>`
-- The push notification snippet must be included on all layouts for mobile and tablet.
-- The pages AppleAdministration, GoogleAdministration and Device_Overview must be connected up to the navigation.
-- You must set up the [apple server](#setting-up-apple-push-notification-server) and [google server](#setting-up-google-cloud-messaging-server) using the documentation bellow
-- The phonegap push plugin must be included in the config.xml, more information can be found [here](#creating-phonegap-app)
-- The widget is connected up to the google settings object. 
-- Encryption module from the appstore must be added to the project.
-
-The application included in the test project can be used as reference.
-
-## Components.json
-All mendix applications that want to utilise phonegap require a components file. The components file describes all the javascript and css files that need to be loaded when the application is loaded on the device. Bellow is an example of the components file that would be required for this module.
+For more information on contributing to this repository visit [Contributing to a GitHub repository](https://world.mendix.com/display/howto6/Contributing+to+a+GitHub+repository)!
+
+## Implementation Guide
+
+### Step 1 - Create an mpk of the PushNotifications module
+
+We need to extract the module from this project before starting with the implementation. Walk through these following steps:
+
+1. Clone this project or download it as ZIP and extract it.
+2. Open the `PushNotifications.mpr` which is located in the `test` directory in the root of the project with a Mendix Modeler.
+3. Right-click on the `PushNotifications` module (inside Project Explorer pane), select `Export module package...` and save the mpk file.
+
+<img src="assets/images/implementation guide/Preparation export Push Notification module.JPG"/>
+
+### Step 2 - Install module dependencies
+
+First, open your existing Mendix project (or create a new one). The Push Notification module has two dependencies: `CommunityCommons` and `Encryption` module. Include these two dependencies by downloading it from the AppStore.
+
+> Note: importing the Encryption module will trigger errors because it contains a reference to a non-existant layout. Fix it by assigning a master layout of `Encryption.ResponsiveLayout_Certificate` page to some other layout (in this specific use case it is not really important which layout is used).
+>
+> <img src="assets/images/implementation guide/Fix Encryption module.JPG"/>
+
+### Step 3 - Import the PushNotification module
+
+Import the created mpk file from Step 1 into your Mendix project. To do this, right-click on an empty space on the Project Explorer pane, select `Import module package...`, choose the mpk file, and add it as a new module.
```

Comment:
Import the mpk file created in Step 1 into your Mendix project

Flows a bit better I think.

Note: right-clicking on the project name at the top of the tree also works and may feel a little more reliable :-)